.github/workflows/cleanup-repo.yaml

```diff
@@ -1,7 +1,7 @@
 ﻿name: "Cleanup Repo"
 on:
   schedule:
-    - cron: '0 7 * * *'
+    - cron: '0 8 * * *'
```

Comment:
<!-- Suggestion: It could be beneficial to include a comment here explaining the reason for the cron schedule change, for better maintainability and clarity for future contributors. -->